style-bibliographique-des-collections-des-shp-ephe-paris.csl

```diff
@@ -466,8 +466,10 @@
       </else-if>
     </choose> 
   </macro>
-  <macro name="isbn">
+  <macro name="ISBN">
+   <group>  
     <text variable="ISBN" prefix=". ISBN : " suffix="."/>
+   </group> 
   </macro>
```

Comment:
Hi Philipp,

thanks! I have added in the layout suffix:
<text macro="ISBN »/>
and it works!

Best,
Vanessa

> Le 6 avr. 2016 à 13:30, Philipp Zumstein <notifications@github.com> a écrit :
> 
> In style-bibliographique-des-collections-des-shp-ephe-paris.csl <https://github.com/citation-style-language/styles/pull/2006#discussion_r58689520>:
> 
> >      <text variable="ISBN" prefix=". ISBN : " suffix="."/>
> > +   </group> 
> >    </macro>
> You don't use this macro here anywhere in your code and therefore you should completely delete it completeley, i.e. delete the 5 lines:
> 
>    <macro name="ISBN">
>    <group>  
>      <text variable="ISBN" prefix=". ISBN : " suffix="."/>
>    </group> 
>    </macro>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/citation-style-language/styles/pull/2006/files/dd68a286710a95d836a57ec0dcaad5038729ac0b..5adf22800a1eb292fc7f34b220327247b2525610#r58689520>